test_cases/arbitrary_jump/FunctionTypes.yaml

```diff
@@ -1,7 +1,8 @@
-description: "Arbitrary jump with function type variable"
+description: Arbitrary jump with function type variable
 issues:
-- id: "SWC-127"
+- id: SWC-127
   count: 1
   locations:
-  - bytecode_offsets: []
-    line_numbers: [26]
+  - bytecode_offsets: {}
```

Comment:
264:1:0 (0x10262f9b836c8ce4a7c2807a00b08e278094414ceeb2b5cbf4365c7ced0fd2f9)

Reply:
There is probably multiple locations that could be considered valid in this test case.